pkg/core/blockchain.go

```diff
@@ -641,8 +657,279 @@ func (bc *Blockchain) SystemFee(t *transaction.Transaction) util.Fixed8 {
 	return bc.GetConfig().SystemFee.TryGetValue(t.Type)
 }
 
+// IsLowPriority flags a trnsaction as low priority if the network fee is less than
+// LowPriorityThreshold
+func (bc *Blockchain) IsLowPriority(t *transaction.Transaction) bool {
+	return bc.NetworkFee(t) < util.NewFixed8FromFloat(bc.GetConfig().LowPriorityThreshold)
+}
+
+// GetMemPool returns the memory pool of the blockchain.
+func (bc *Blockchain) GetMemPool() MemPool {
+	return bc.memPool
+}
+
+// Verify verifies whether a transaction is bonafide or not.
+// Golang implementation of Verify method in C# (https://github.com/neo-project/neo/blob/master/neo/Network/P2P/Payloads/Transaction.cs#L270).
+func (bc *Blockchain) Verify(t *transaction.Transaction) bool {
```

Comment:
what you think about provide `error` unless `bool`
with error and pkg/errors you can provide reason ("why transaction not valid").
that's would be very useful in development and debug mode, what you think?

Reply:
Good idea! 👍